README.rst

```diff
@@ -91,29 +91,21 @@ Step 3: Set environment variables
 
 2. If you use Smart Gateway, or want to ingest directly from a realm other than us0, then you must set at least one endpoint variable. (For environment variables, SignalFx defaults to the us0 realm. As a result, if you are not in the us0 realm, you may need to set your environment variables.) There are two options: 
 
-**Option 1**
-
-You can update ``SIGNALFX_ENDPOINT_URL`` where both metrics and traces will be sent to the gateway address. Note that the path ``/v1/trace`` will be automatically added to the endpoint for traces.
-
-.. code:: bash
 
-    SIGNALFX_ENDPOINT_URL=http://<my_gateway>:8080
-    
-
-**Option 2**
+**Option 1**
 
-You can update ``SIGNALFX_ENDPOINT_URL`` and ``SIGNALFX_METRICS_URL`` where traces will be sent to the gateway and metrics will go directly to the ingest endpoint, respectively. 
+Set ``SIGNALFX_TRACING_URL`` and ``SIGNALFX_METRICS_URL`` to configure where traces and metrics will be sent to. The following example will send traces to the gateway and metrics directly to the ingest endpoint. 
```

Comment:
This is the actual doc correction that's required.  Do you see any reason not to list the dynamic endpoint url as option 2 until the collector path is ironed out?  I can understand removing the gateway-specific at obsolescence so it's not a blocker but a bit unclear why we'd remove it entirely atm.

Reply:
I think there is a bug somewhere that prevents tracing/metrics from working when using SIGNALFX_TRACING_URL only. I ran into two customers who faced the issue and they were exporting directly to SFX ingest. The reason to not mention it as option 2 is that I don't understand why didn't work for some people and until I figure out the root cause and fix the bug, I don't want to recommend it anymore. 

So, we can add it back as an option once I fix or rule out the bug I suspect is lurking around in the project somewhere.